examples/apps/heart_disease_eval/workload/heart_disease_evaluation_logic.cpp

```diff
@@ -16,36 +16,37 @@
 #include <string>
 #include <cmath>
 #include "heart_disease_evaluation_logic.h"
+#include "utils.h"
 
 // Calculate risk of having heart disease given input parameters.
 
 HeartDiseaseEvalLogic::HeartDiseaseEvalLogic() {}
 
 HeartDiseaseEvalLogic::~HeartDiseaseEvalLogic() {}
 
-template<typename Out>
-void split(const std::string &str, char delim, Out result) {
-        std::size_t current, previous = 0;
+template<typename Out>	
```

Comment:
Remove the several trailing spaces at the end of this line and other lines in this file.
Some of these lines were not changed.